src/actions/breakpoints/addBreakpoint.js

```diff
@@ -152,10 +157,15 @@ export function addBreakpoint(
   location: SourceLocation,
   { condition, hidden }: addBreakpointOptions = {}
 ) {
-  const breakpoint = createBreakpoint(location, { condition, hidden });
-  return ({ dispatch, getState, sourceMaps, client }: ThunkArgs) => {
+  return async ({ dispatch, getState, sourceMaps, client }: ThunkArgs) => {
     recordEvent("add_breakpoint");
 
+    if (location.column === undefined) {
+      location = getFirstPausePointLocation(getState(), location);
```

Comment:
Did we want this behavior toggled based on the column breakpoint feature flag?

Reply:
Probably a good idea; I'll add it!